src/ensembl/src/shared/components/selected-species/FocusableSelectedSpecies.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import SelectedSpeciesDisplayName from './SelectedSpeciesDisplayName';
+
+import styles from './FocusableSelectedSpecies.scss';
+
+import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
+
+export type Props = {
+  species: CommittedItem;
+  isActive: boolean;
+  onClick: (genomeId: string) => void;
+  onMouseEnter?: () => void;
+  onMouseLeave?: () => void;
+  className?: string;
+};
+
+const SpeciesTab = (props: Props) => {
+  const handleMouseEnter = () => {
+    props.onMouseEnter && props.onMouseEnter();
+  };
+
+  const handleMouseLeave = () => {
+    props.onMouseLeave && props.onMouseLeave();
+  };
+
+  const handleClick = () => {
+    if (!props.isActive) {
+      props.onClick(props.species.genome_id);
+    }
+  };
+
+  const className = classNames(styles.species, props.className, {
+    [styles.speciesActive]: props.isActive
+  });
+  const nameClass = props.isActive ? styles.nameActive : styles.name;
```

Comment:
What does `nameActive` mean here?

Reply:
A species pill contains species name and assembly name. The class for species name is here called `name`; the class for assembly name is Species name is `assembly`. The pill can also be in the "active" (selected, focused) or "inactive" (normal) state. Thus, the `nameActive` class is the class that is applied to the species name in the active pill.